docs/UserGuide.md

```diff
@@ -532,15 +537,40 @@ Format: `list me`
 Generates a list of internships that have requirements that matches your current set of skills.
 
 Format: `match`
-- Matching done is case-insensitive. e.g. `Python` will match `python`
-- Internships with job titles matching at least one `SKILL` will be returned.
-- Only full words will be matched. e.g. `React` will not match `React Native`
+- Only the `Requirement` field in the internships will be searched for.
+- Only profile items with `Skill` category will be used for matching.
+- An Internship with at least one `Requirement` matching any one `Skill` in the profile list will be
+considered as a successful match. e.g. Say you have a profile item which is of `Skill` category and titled `Python`, any
+internship that has a `Requirement` of `Python` will be successfully matched.
+- Matched internships can have `Requirements` that do not match with the profile `Skills`, since a match is found when
+an internship has **at least one and not all** `Requirements` that matches with the profile `Skills`.
+- Matching done is case-insensitive. e.g. `Python` will match `python`.
+- Only the full sentence will be matched. e.g. `Machine` will not match `Machine Learning`.
```

Comment:
phrase rather than sentence?

Reply:
Thanks, been finding a way to make this clearer